pallet/src/lib.rs

```diff
@@ -198,7 +216,7 @@ pub mod pallet {
             {
                 if session_for_validators_change <= session {
                     let validators = Pallet::<T>::validators().expect(
-                        "Validators also should be Some(), when session_for_validators_change is",
+                        "When session_for_validators_change is Some() validators should also be Some()",
```

Comment:
We could enforce this on type levels by storing the validators and their session in one `Option<(u32, Vec<T::AccountId>)>`

Reply:
commited by mistake, reverted

Reply:
Yeah, you can add a ticket.

Reply:
https://cardinal-cryptography.atlassian.net/browse/AZ-271